### PR TITLE
chore(mobile): patch download > includeEmbeddedVideos user preferences

### PR DESCRIPTION
**Pull request by alextran1502**

### CODE REVIEW

File: mobile/lib/utils/openapi_patching.dart [hunk 1]
```diff
@@ -7,6 +7,11 @@ dynamic upgradeDto(dynamic value, String targetType) {
         if (value['rating'] == null) {
           value['rating'] = RatingResponse().toJson();
         }
+
+        if (value['download']['includeEmbeddedVideos'] == null) {
```
Comment (jrasm91) [inline mobile/lib/utils/openapi_patching.dart:11]: why is this needed? It has a default value of false already.

```dart
class DownloadResponse {
  /// Returns a new [DownloadResponse] instance.
  DownloadResponse({
    required this.archiveSize,
    this.includeEmbeddedVideos = false,
  });
```

Comment (alextran1502) [reply mobile/lib/utils/openapi_patching.dart:11]: So when you access the older server, the response from the old server doesn't have `includeEmbeddedVideos`

**Review by jrasm91 — APPROVED** [review]
